.github/workflows/infrastructure.yml

```diff
@@ -261,6 +261,7 @@ jobs:
             --storage-size ${{ vars.DB_STORAGE_SIZE_GB }} \
             --sku-name ${{ env.DB_SKU }} \
             --tier ${{ env.DB_TIER }} \
+            --version ${{ vars.DB_VERSION }} \
```

Comment:
is this set to 8.0.36 where the vars comes from?

Reply:
Yes, it comes from GitHub